migration/sql-files/061-replace-index-space-name.sql

```diff
@@ -1,4 +1,14 @@
 -- drop existing unique index
 DROP INDEX spaces_name_idx;
+-- rename duplicate spaces in existence and keep only one as it was
+UPDATE spaces SET name = CONCAT(lower(name), '-renamed')
```

Comment:
we could eventually (lower(name), '-', uuid_generate_v4()) or even get the last digits of its own id.